gef-ui/webpack.common.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     govukFrontend: './scripts/govuk-frontend.js',
     jsEnabled: './scripts/js-enabled.js',
     correspondenceAddress: './scripts/correspondence-address.js',
+    disableFormSubmitOnSubmission: '../libs/common/src/ui-scripts/disable-form-submit-on-submission.js',
```

Comment:
Just want to double check: Webpack is only ever called in the workspaces directory set up right - this should never be called within a service when it wont have this to reference

Reply:
This should be fine as we only call webpack config manually (via `npm run build` for example). We don't call this when we build automagically in ci/cd as we need to update the SHA to allow the js to run :)